proc/ascent-entry/ES1.md

```diff
@@ -0,0 +1,44 @@
+# Ascent-Entry: ES1
+
+## SLS Flies to Kerbin orbit
+
+#### Starting Assumptions
+
+* SLS on pad at space center
+* All setup procedures have been completed
+
+Unless otherwise specified, all actions below are completed by SLS. Target values are sometimes provided. Acceptable error values are provided in brackets following the target value.
+
+#### Begin Ascent-Entry Event Sequence 1
+
+1. Core Stage (CS) Ignition
+
+2. Solid Rocket Booster (SRB) Ignition (cross strapped if applicable in KSP)
+
+3. Release Umbilicals once CS and SRB both reach 100% throttle
+
+4. Start attitude control routine in parallel, maintain 0 degree inclination [0.1]
+
+   A. Use closed loop proportional controller (guidance) to maintain attitude, stay on trajectory
+
+5. When SRB fuel under 5% of max value, throttle CS down to 67%
```

Comment:
I would recommend not hard-coding these values for the throttling. I would instead recommend having your main engines smoothly throttle down to maintain a tolerable thrust-to-weight ratio while the SRBs are attached, then throttle them back up once SRBs are jettisoned (possibly to the same thrust-to-weight ratio).

Reply:
That makes a lot of sense. Thanks for the approval, I'm gonna go ahead and merge it for now so other teams can see the draft. We can (and will!) update this doc as we get smarter about our procedures, and test things through `krpc` / `ksp`.